src/portal/lib/src/service/project.service.ts

```diff
@@ -23,7 +23,7 @@ import { RequestQueryParams } from "./RequestQueryParams";
  */
 export abstract class ProjectService {
   /**
-   * Get Infomations of a specific Project.
+   * Get Informations of a specific Project.
```

Comment:
This still feels weird to me. What about

```suggestion
   * Get Information about a specific Project.
``

Reply:
Thank you for your reminding.🐯